fingertip_keyboard_input/mouse_and_keyboard_control.py

```diff
@@ -334,59 +345,109 @@ def get_filtered_center(center, t):
                     if (time_stamp - time_start > 1):
                         time_start = time_stamp
                         if right_gesture == "one": #switch to previous app
-                            pyautogui.hotkey('command', 'tab')
+                            # pyautogui.hotkey('command', 'tab')
+                            # added Windows shortcut commands
+                            if system_type == "Mac":
+                                pyautogui.hotkey('command', 'tab')
+                            elif system_type == "Windows":
+                                pyautogui.hotkey('alt', 'tab')
                         elif right_gesture == "two": #browse windows
-                            pyautogui.hotkey('ctrl', 'up')
+                            # pyautogui.hotkey('ctrl', 'up')
+                            if system_type == "Mac":
+                                pyautogui.hotkey('ctrl', 'up')
+                            elif system_type == "Windows":
+                                pyautogui.hotkey('ctrl', 'alt', 'tab')
```

Comment:
I wonder if there's a way to "click" on a selected window without actually switching to the "mouse" mode. I may be misinterpreting this functionality though.

Reply:
Do you mean we could get to the "window" and "browser" mode during the default "keyboard" mode instead of getting into "mouse" mode fist?

Reply:
solved via meeting/slack. Removing functionality.